pkg/opavalidation/opavalidation.go

```diff
@@ -15,34 +15,69 @@ import (
 	"github.com/open-policy-agent/opa/rego"
 	"github.com/open-policy-agent/opa/topdown"
 	"github.com/open-policy-agent/opa/types"
+	"github.com/sirupsen/logrus"
 )
 
-// Run is a ValidateRego() wrapper that prints resulting actionItems. This is
+const (
+	DefaultKubeObjectNamespace = "notset" // The namespace to use if one is unspecified
+)
+
+// Run is a ValidateRego() wrapper that validates and prints resulting actionItems. This is
 // meant to be called from a cobra.Command{}.
-func Run(regoFileName, objectFileName string, insightsInfo fwrego.InsightsInfo, objectNamespaceOverride string) error {
+func Run(regoFileName, objectFileName string, insightsInfo fwrego.InsightsInfo, objectNamespaceOverride string) (actionItems, error) {
```

Comment:
The returned `actionItems` are not used anywhere. any reason why it was added as a returned slice.

Reply:
At one point I implemented this but ended up not needing it, however I left it in place to avoid the actionItems being "lost" or "trapped" inside of the `Run*` functions. For example, if another part of Insights used this package in the future and thatcaller to `Run` or `RunBatch` wanted to inspect the output of inner; child functions, this wouldn't be possible if `Run*` did not return its results. @makoscafee What are your thoughts - is this good package API, or something that should be removed for now?

Reply:
We had a discussion in Slack and the outcome is to leave this in, and potentially break it out so the Run* functions don't directly print in the future.